tt_metal/hw/inc/wormhole/eth_l1_address_map.h

```diff
@@ -65,6 +70,9 @@ struct address_map {
 
     static_assert((ERISC_L1_UNRESERVED_BASE % 32) == 0);
 
+    static constexpr std::int32_t ERISC_MEM_BANK_TO_NOC_SCRATCH = (ERISC_L1_KERNEL_CONFIG_BASE + ERISC_L1_KERNEL_CONFIG_SIZE + 31) & ~31; //Same as ERISC_L1_UNRESERVED_BASE
```

Comment:
You should move `ERISC_L1_UNRESERVED_BASE` to a different address right? Otherwise, wouldn't this conflict with user data that starts at `ERISC_L1_UNRESERVED_BASE`?

Reply:
We are using this space temporarily, the host writes the data to this L1 space during runtime build. Firmware would then copy the data from L1 to the local memory during initialization. Once firmware initialization is done, the space can be used for user data.

Reply:
can you add a comment explaining this, just so people that aren't familiar with this don't run into the same confusion